django/.pa11yci

```diff
@@ -3,16 +3,15 @@
         "runners": ["axe", "htmlcs"],
         "chromeLaunchConfig": {
             "args": ["--no-sandbox"]
-        }
+        },
+        "hideElements": [
```

Comment:
What you have here will work, but is it maybe worth considering alternatives such as allowing testing within the hidden elements by either using 
```
       "ignore": [
          "color-contrast"
        ]
```
or using sed to manipulate the css to hide the background image (using the fallback colour only) in .gitlab-ci.yml before pa11y-ci is run?

Reply:
thanks, that's helpful. have added the ignore for color contrast. it passed Wave and the color.review said it was suitable but no matter what I set it to (even black) pa11y still complained. it's becoming quite a frustrating tool to use these days! thanks again for the tip